docs/tables/scaleway_baremetal_server.md

```diff
@@ -0,0 +1,55 @@
+# Table: scaleway_baremetal_server
+
+An bare metal provides dedicate server to run your applications on.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  status,
+  updated_at,
+  created_date,
+  zone,
+  project,
+  organization
+from
+  scaleway_baremetal_server;
+```
+
+### List stopped baremetal servers
```

Comment:
```suggestion
### List stopped bare metal servers
```

I think we should use `bare metal` instead of `baremetal` since majority of the documents pertaining to scaleway bare metal servers use the former.

https://www.scaleway.com/en/bare-metal-definition/

Reply:
yes you're right, I wanted to be homogeneous